File: proj/code-repair/mycode/clean_dataset/patch-single-line/quixbugs-python/gpt-neo-1.3/kth.py
--- 
+++ 
@@ -9,6 +9,6 @@
     if k < num_less:
         return kth(below, k)
     elif k >= num_lessoreq:
-        return kth(above, k)
+        return kth(above, k - num_lessoreq)
     else:
         return pivot

File: proj/code-repair/mycode/clean_dataset/patch-single-line/quixbugs-java/codex-sh/kth.java
--- 
+++ 
@@ -16,7 +16,7 @@
     if (k < num_less) {
         return kth(below, k);
     } else if (k >= num_lessoreq) {
-        return kth(above, k);
+        return kth(above, k - num_lessoreq);
     } else {
         return pivot;
     }
